connection/cluster/ClusterClient.java

```diff
@@ -63,31 +64,25 @@
     private final ConcurrentMap<String, ClusterDatabase> clusterDatabases;
     private boolean isOpen;
 
-    public ClusterClient(Set<String> addresses, TypeDBCredential credential) {
-        this(addresses, credential, ClusterServerClient.calculateParallelisation());
+    public ClusterClient(Set<String> initAddresses, TypeDBCredential credential) {
```

Comment:
Rename the variable to emphasise that this variable refers to the "initial set of addresses" given by the user. This set may be partially out of date, and should be made up-to-date during the client-cluster mediation process.